stubs/generate_pyi.py

```diff
@@ -62,7 +62,7 @@ def _build_base_config_pyi(filename, base_pyi):
         base_pyi += "".join(lines[begin_line:end_line])
 
         if '"""' not in lines[end_line - 1]:
-            base_pyi += '\t\t""""""\n'
+            base_pyi += '\t\t""""""\n'.replace("\t", "    ")
```

Comment:
Just a question, why `.replace()` but not just `'        """"""\n'`

Reply:
Just a visual preference, for me is more readable and easy to understand the ammount of spaced like this. 

But anyone is fine for me.